WordPress/src/main/res/layout/restore_list_subheader_item.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/item_container"
+    android:layout_width="match_parent"
+    android:layout_height="?attr/listPreferredItemHeight">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/subheader"
+        style="@style/Jetpack.SecondaryHeader"
+        android:layout_marginTop="13dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        tools:text="Choose the items to restore:"
```

Comment:
Minor (🔍): This `tools:text` is not necessary since `android:text` exists as well. Consider removing that.

PS: The same applies for the backup download corresponding layout file.

Reply:
in commit 34fd983
I'll take care of download during consolidation - there will only be 1 version of the file